src/components/RequestsFilters/RequestsFilters.test.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import RequestsFilters from './RequestsFilters';
+
+jest.mock('@folio/stripes/components', () => ({
+  AccordionSet: jest.fn(({ children }) => <div>{children}</div>),
+  Accordion: jest.fn(({ displayClearButton, name, children, onClearFilter }) => (
+    <div>
+      <div>{displayClearButton ? <button type="button" onClick={() => onClearFilter()} data-testid={`clear-${name}`}>Clear</button> : null}</div>
+      <div data-testid={`accordion-${name}`}>{children}</div>
+    </div>
+  )),
+  FilterAccordionHeader: jest.fn(() => <div>FilterAccordionHeader</div>),
+}));
+
+jest.mock('@folio/stripes/smart-components', () => ({
+  CheckboxFilter: jest.fn(() => <div>CheckboxFilter</div>),
+  MultiSelectionFilter: jest.fn(() => <div>MultiSelectionFilter</div>),
+}));
```

Comment:
We have separate files with mocks - `test/jest/__mock__`.
You can add your mocks there if those mocks do not exist yet.